components/Alerts/alerts.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+    <link rel="stylesheet" href="alerts.css">
+</head>
+
+<body>
+    <div class="alert">
+        <div class="alert primary-alert">A primary alert example with a text link. <span class="link link-active">Check
+                it out</a><span></div>
+        <div class="alert secondary-alert">A secondary alert example.</div>
+        <div class="alert success-alert"><span class="m material-icons">check_circle</span>A success alert example.
+        </div>
+        <div class="alert danger-alert"><span class="m material-icons">dangerous</span>A danger alert example.</div>
+        <div class="alert warning-alert"><span class="m material-icons">warning</span> A warning alert example.</div>
+        <!-- carbon code link -->
+        <iframe 
+            src="https://carbon.now.sh/embed?bg=rgba%28245%2C166%2C35%2C0.78%29&t=3024-night&wt=none&l=css&width=680&ds=true&dsyoff=3px&dsblur=13px&wc=true&wa=true&pv=14px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253C%21--Primary%2520Alert--%253E%250A%253Cdiv%2520class%253D%2522alert%2520primary-alert%2522%253EA%2520primary%2520alert%2520example%2520with%2520a%2520text%2520link%2520%253Cspan%2520class%253D%2522link%2520link-active%2522%253ECheck%2520it%2520out%2520%253C%252Fa%253E%253Cspan%253E%253C%252Fdiv%253E%250A%253C%21--Primary%2520Alert--%253E%250A%253C%21--Secondary%2520Alert--%253E%250A%253Cdiv%2520class%253D%2522alert%2520secondary-alert%2522%253EA%2520secondary%2520alert%2520example.%253C%252Fdiv%253E%250A%253C%21--Secondary%2520Alert--%253E%250A%253C%21--Success%2520Alert--%253E%250A%253Cdiv%2520class%253D%2522alert%2520success-alert%2522%253E%253Cspan%2520class%253D%2522m%2520material-icons%2522%253Echeck_circle%253C%252Fspan%253EA%2520success%2520alert%2520example.%253C%252Fdiv%253E%250A%253C%21--Success%2520Alert--%253E%250A%253C%21--Danger%2520Alert--%253E%250A%253Cdiv%2520class%253D%2522alert%2520danger-alert%2522%253E%253Cspan%2520class%253D%2522m%2520material-icons%2522%253Edangerous%253C%252Fspan%253EA%2520danger%2520alert%2520example.%253C%252Fdiv%253E%250A%253C%21--Danger%2520Alert--%253E%250A%253C%21--Warning%2520Alert--%253E%250A%253Cdiv%2520class%253D%2522alert%2520warning-alert%2522%253E%253Cspan%2520class%253D%2522m%2520material-icons%2522%253Ewarning%253C%252Fspan%253EA%2520warning%2520alert%2520example.%253C%252Fdiv%253E%250A%253C%21--Warning%2520Alert--%253E"
+            style="width: 800px; height: 273px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+    </div>
+
```

Comment:
The inedntation i.e spacing in the file could be improved.

Reply:
updated the structure and spacing .